### DISCUSSION TITLE
Add support for feed search service to find RSS feeds links

### DISCUSSION THREAD

**Discussion by TheOneImprove** [jocmp/capyreader#527] [Feature Requests]
I suggest adding the ability to search [third party RSS feeds provider](https://feedsearch.dev/) for RSS feeds links.

**Comment by jocmp** [comment]
Hey @TheOneImprove I like the idea. Let me think on how I could add this. The biggest issue might be whether or not https://feedsearch.dev allows third-party integrations as a part of their terms of service, but I'll look into it.
